code/esp_code/tcp_client.c

```diff
@@ -109,59 +150,22 @@ static void tcp_client_task(void *pvParameters) {
         ESP_LOGI(TAG, "Successfully connected");
 
         int n = 0;
-        esp_err_t ret;
-
-        // Configuration for the SPI bus
-        spi_bus_config_t buscfg = {.mosi_io_num = GPIO_MOSI,
-                                   .miso_io_num = GPIO_MISO,
-                                   .sclk_io_num = GPIO_SCLK};
-
-        // Configuration for the SPI slave interface
-        spi_slave_interface_config_t slvcfg = {
-            .mode = 1,
-            .spics_io_num = GPIO_CS,
-            .queue_size = 3,
-            .flags = 0,
-            .post_setup_cb = my_post_setup_cb,
-            .post_trans_cb = my_post_trans_cb};
-
-        // Configuration for the handshake line
-        gpio_config_t io_conf = {.intr_type = GPIO_INTR_DISABLE,
-                                 .mode = GPIO_MODE_OUTPUT,
-                                 .pin_bit_mask = (1 << GPIO_HANDSHAKE)};
-
-        // Configure handshake line as output
-        gpio_config(&io_conf);
-        // Enable pull-ups on SPI lines so we don't detect rogue pulses when no
-        // master is connected.
-        gpio_set_pull_mode(GPIO_MOSI, GPIO_PULLUP_ONLY);
-        gpio_set_pull_mode(GPIO_SCLK, GPIO_PULLUP_ONLY);
-        gpio_set_pull_mode(GPIO_CS, GPIO_PULLUP_ONLY);
-
-        // Initialize SPI slave interface
-        ret = spi_slave_initialize(HSPI_HOST, &buscfg, &slvcfg, 1);
-        if (ret != ESP_OK) {
-            ESP_LOGE(TAG, "Failed to initialize spi slave");
-            break;
-        }
-
-        memset(recvbuf, 0, 129);
-        spi_slave_transaction_t t = {0};
-        memset(&t, 0, sizeof(t));
-
+        
         while (1) {
             // Clear receive buffer,
-            memset(recvbuf, 0x00, 129);
+            memset(recvbuf, 0xA5, 129);
```

Comment:
Is it documented why these values are used?

Reply:
it was used for debugging